harness/determined/common/experimental/checkpoint/_checkpoint.py

```diff
@@ -163,7 +164,7 @@ def download(self, path: Optional[str] = None) -> str:
         # export api, we need to start writing a different format (the user-defined metadata only),
         # and we need to start writing it at upload time, not download time.
         #
-        # see https://determined-ai.slack.com/archives/CSLAGUF3M/p1642188450030600?thread_ts=1642115346.016000&cid=CSLAGUF3M
+        # see https://rb.gy/vaxl7a
```

Comment:
IMO we should avoid having links to private slack in the repo.  I know it's already landed, but can we copy or paraphrase the context here instead?